pkgs/tools/networking/pihole-ftl/version.patch

```diff
@@ -0,0 +1,20 @@
+diff -uprN FTL-5.3.4/src/version.h.in FTL-5.3.4.cust/src/version.h.in
+--- FTL-5.3.4/src/version.h.in	2020-12-25 19:23:09.000000000 +0100
++++ FTL-5.3.4.cust/src/version.h.in	2020-12-26 20:59:22.279453551 +0100
+@@ -1,11 +1,11 @@
+ #ifndef VERSION_H
+ #define VERSION_H
+ 
+-#define GIT_VERSION "@GIT_VERSION@"
+-#define GIT_DATE "@GIT_DATE@"
+-#define GIT_BRANCH "@GIT_BRANCH@"
+-#define GIT_TAG "@GIT_TAG@"
+-#define GIT_HASH "@GIT_HASH@"
++#define GIT_VERSION "v5.3.4"
```

Comment:
Please use substituteInPlace to dynamically set those based on the version variable. Can we supply some flag to set those?